spec/lib/hmrc_interface/request/submission_spec.rb

```diff
@@ -0,0 +1,222 @@
+require "rails_helper"
+
+RSpec.describe HmrcInterface::Request::Submission do
+  subject(:instance) { described_class.new(client, use_case, filter) }
+
+  let(:client) { HmrcInterface.client }
+  let(:use_case) { submission.use_case }
+
+  let(:filter) do
+    {
+      nino: submission.nino,
+      start_date: submission.period_start_at,
+      end_date: submission.period_end_at,
+      first_name: submission.first_name,
+      last_name: submission.last_name,
+      dob: submission.dob,
+    }
+  end
+
+  let(:submission) { create(:submission, :for_sandbox_applicant) }
+
+  # Remove or keep?
```

Comment:
I would say maybe remove, as we wouldn't ever want to be actually making unstubbed calls in the CI pipeline?